api/proto/ankaios.proto

```diff
@@ -130,29 +130,81 @@ enum AddCondition {
   ADD_COND_FAILED = 2; /// The workload has exited with an error or could not be started.
 }
 
+
+message ExecutionState {
+    string additionalInfo = 1;
```

Comment:
Is there any description for the `additionalInfo`?

Reply:
```suggestion
    string additionalInfo = 1; /// The additional info contains more detailed information from the runtime regarding the execution state.
```